doc/develop/fixing_data.rst

```diff
@@ -329,27 +329,42 @@ strictness to the highest:
   be sure that future users will not be distracted by inoffensive warnings.
 
 
-Natively supported non-CMIP datasets
-====================================
+.. _fixing_native_datasets:
 
-Some fixed datasets and native models formats are supported through
-the ``native6`` project or through a dedicated project.
+Natively supported datasets
```

Comment:
Could you move this section, i.e. the list of datasets supported datasets to https://esmvaltool--1559.org.readthedocs.build/projects/ESMValCore/en/1559/quickstart/find_data.html#datasets-in-native-format? I think it does not make sense that people who just want to use some ERA5 data need to dive into the development documentation.

Reply:
Done!